engine-rest/engine-rest-openapi/src/main/templates/paths/engine/get.ftl

```diff
@@ -0,0 +1,26 @@
+{
+  <@lib.endpointInfo
+      id = "getEngineNames"
```

Comment:
It seems that the Java method is called `getProcessEngineNames`, this is only nice to have, it is not required, the only condition is to be unique id, 
```suggestion
      id = "getProcessEngineNames"
```